# Table of Content

- [Table of Content](#table-of-content)
- [0-General](#0-general)
  - [0-Introduction](#0-introduction)
  - [0-Objective](#0-objective)
  - [0-Analysis](#0-analysis)
  - [0-Data Dictionary](#0-data-dictionary)
  - [0-Acknowledgements](#0-acknowledgements)
- [1-Sample Size Calculation](#1--sample-size-calculation)

# 0-General
[Back to Table of Content](#table-of-content)

# Sample Size Calculation for MindSpaceOne 2.0

## Introduction
[Back to Table of Content](#table-of-content)

This sample size calculation aims to estimate the minimum number of participants required per group to detect differences in the primary outcome—subjective relaxation (MDBF "Ruhe" subscale)—after a 6-week intervention in the MindSpaceOne 2.0 RCT. The design includes four arms:

1. HYPNOSE + BERGAMOTTE
2. HYPNOSE + LAVENDEL
3. HYPNOSE-only
4. Pseudo-Control (neither hypnotherapy nor aroma)

A one-way ANCOVA will be used to assess between-group differences at T2 (post-intervention), controlling for baseline MDBF values.

### Reference Study Data: MindSpaceOne 1.0

Based on results from the previous MindSpaceOne 1.0 study:
- Cohen's d for HYPNOSE+AROMA vs. Control ≈ 0.38
- Cohen's d for HYPNOSE-only vs. Control ≈ 0.33

To remain conservative while still allowing for a slightly broader recruitment base, we adopt Cohen’s f = 0.175 as the expected effect size for group differences. This value is slightly below the midpoint of the observed range (d ≈ 0.35–0.38), but still represents a realistic estimate of detectable between-group variance. Given the intensity and length of the intervention (6 weeks, 24 sessions), a dropout rate of 35% is assumed to reflect potential participant burden in this preventive, non-clinical sample.

# 1- Sample Size Calculation
[Back to Table of Content](#table-of-content)

In [17]:
# Load required package
if (!require(pwr)) install.packages("pwr")
library(pwr)

### === INPUT SECTION ===

k <- 4                        # number of groups in the initial ANOVA
cohen_f_raw <- 0.175         # conservative effect size estimate (based on PP d ≈ 0.35–0.38)
r_squared <- 0.20            # variance explained by baseline covariate (ANCOVA)
alpha <- 0.05                # significance level (two-sided)
power <- 0.80                # desired statistical power
dropout_rate <- 0.30         # expected dropout rate due to study length/frequency

# === EFFECT SIZE ADJUSTMENT FOR ANCOVA ===
cohen_f_adj <- cohen_f_raw / sqrt(1 - r_squared)

# === RAW SAMPLE SIZE CALCULATION ===
result <- pwr.anova.test(k = k, f = cohen_f_adj, sig.level = alpha, power = power)
n_raw <- ceiling(result$n)  # raw sample size per group before dropout
cat("Required sample size per group (no dropout):", n_raw, "\n")

# === DROPOUT ADJUSTMENT ===
n_corrected <- ceiling(n_raw / (1 - dropout_rate))  # adjusted for 35% dropout
cat("Required sample size per group (with 35% dropout):", n_corrected, "\n")

# === COMPENSATION FOR SUBGROUP SPLIT ===
# Two of the four groups will later be split into subgroups,
# so they require double the sample size to retain statistical power in the split.
# We assume groups 1 and 2 need to be doubled.

group_sizes <- c(
  n_corrected * 2,  # Group 1 (to be split)
  n_corrected * 2,  # Group 2 (to be split)
  n_corrected,      # Group 3
  n_corrected       # Group 4
)

# === TOTAL SAMPLE SIZE CALCULATION ===
total_n <- sum(group_sizes)
cat("Adjusted total sample size with subgroup compensation:", total_n, "participants\n")

Required sample size per group (no dropout): 73 
Required sample size per group (with 35% dropout): 105 
Adjusted total sample size with subgroup compensation: 630 participants


# 2-Conclusion
[Back to Table of Content](#table-of-content)

The objective of this analysis was to estimate the minimum sample size required per group to ensure adequate statistical power for detecting group differences in subjective relaxation as measured by the “Ruhe” subscale of the Mehrdimensionaler Befindlichkeitsfragebogen (MDBF) following a 6-week digital intervention. The planned randomized controlled trial includes four parallel arms and uses a one-way ANCOVA model for the primary analysis, with baseline scores included as covariates to improve statistical power and reduce residual variance.

The expected effect size was based on outcome data from the previous MindSpaceOne 1.0 study, where Cohen’s d ranged between 0.33 and 0.49 for comparisons between hypnotherapeutic interventions and control. A conservative average was adopted and translated to a Cohen’s f of 0.175. Assuming baseline values explain 20% of the outcome variance (R² = 0.20), this results in an adjusted Cohen’s f of approximately 0.196 for the ANCOVA model.

Using this adjusted effect size and targeting a conventional power of 80% with a two-sided significance level of 0.05, the estimated sample size per group is 73 participants (without dropout). To account for an anticipated dropout rate of 35%—which reflects the expected burden of a high-frequency 6-week digital intervention—the adjusted sample size increases to 105 participants per group.

Importantly, two of the four study arms will be split into subgroups for further follow-up analyses after the main intervention phase. To ensure sufficient power for these subsequent subgroup comparisons, the initial sample size in those two arms must be doubled. As a result, the final recommended total sample size is 630 participants across all groups.

In conclusion, recruiting 630 participants will ensure that MindSpaceOne 2.0 is sufficiently powered to detect meaningful group differences in subjective relaxation, accommodate expected dropout, and support additional planned subgroup analyses within the trial design.